{"name": "test_tip", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Error is list index out of range' is True\n +  where 'Error is list index out of range' = tip_editing()\n +    where tip_editing = <logic.setup_logic.Setup object at 0x0000016647F84490>.tip_editing\n +      where <logic.setup_logic.Setup object at 0x0000016647F84490> = <test_setup.TestSetUpFlow object at 0x0000016647F6AB00>.setup", "trace": "self = <test_setup.TestSetUpFlow object at 0x0000016647F6AB00>\n\n    def test_tip(self):\n>       assert self.setup.tip_editing() is True\nE       AssertionError: assert 'Error is list index out of range' is True\nE        +  where 'Error is list index out of range' = tip_editing()\nE        +    where tip_editing = <logic.setup_logic.Setup object at 0x0000016647F84490>.tip_editing\nE        +      where <logic.setup_logic.Setup object at 0x0000016647F84490> = <test_setup.TestSetUpFlow object at 0x0000016647F6AB00>.setup\n\nAutomation-logic\\suits\\test_setup\\test_setup.py:29: AssertionError"}, "start": 1743594830645, "stop": 1743594857762, "uuid": "91506f23-d908-4679-9e20-4853bb890489", "historyId": "e4f50cb10e26e558ca28fa337e2a83bf", "testCaseId": "e4f50cb10e26e558ca28fa337e2a83bf", "fullName": "suits.test_setup.test_setup.TestSetUpFlow#test_tip", "labels": [{"name": "feature", "value": "Developer 1 Report"}, {"name": "parentSuite", "value": "suits.test_setup"}, {"name": "suite", "value": "test_setup"}, {"name": "subSuite", "value": "TestSetUpFlow"}, {"name": "host", "value": "LAPTOP-KKMKONCC"}, {"name": "thread", "value": "23228-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "suits.test_setup.test_setup"}]}